files/initramfs-tools/arista-convertfs.j2

```diff
@@ -82,6 +82,24 @@ run_cmd() {
     fi
 }
 
+create_varlog_file() {
```

Comment:
I see. This function is Arista specific because in the process of Arista aboot installation, there is no tool to create an ext4 partition, and it must be postpone to initramfs.